packages/lexical-clipboard/src/clipboard.ts

```diff
@@ -519,7 +519,7 @@ function $copyToClipboardEvent(
 ): boolean {
   if (data === undefined) {
     const domSelection = getDOMSelection(editor._window);
-    if (!domSelection) {
+    if (!domSelection || domSelection.isCollapsed) {
```

Comment:
This is always true, you forgot to call the method. I'm surprised that eslint isn't complaining about this.

```suggestion
    if (!domSelection || domSelection.isCollapsed()) {
```

Reply:
I don't have knowledge about writing tests. But surely I'll learn and write those. It would be really helpful if you can guide me a little here.

Reply:
https://lexical.dev/docs/api/#running-tests (note that the e2e tests assume you are running the playground in another tab)

I'd suggest taking a look at the other clipboard related tests to see how they work. Tests for this would probably belong in this file: https://github.com/facebook/lexical/blob/main/packages/lexical-playground/__tests__/e2e/CopyAndPaste/lexical/CopyAndPaste.spec.mjs

Reply:
if you put something like
`if (!domSelection || domSelection.isCollapsed) {
      return false;
    }`
    you're able to check weather the node is empty or not and so, I think it works fine as in the video also above it is working fine.